android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -367,7 +370,6 @@ private void initializePlayerControl() {
 
         // Setting the player for the playerControlView
         playerControlView.setPlayer(player);
-        playerControlView.show();
```

Comment:
here also you are changing the behavior is it really expected ?
Maybe a new prop can be added to change behavior

Reply:
I think this show() is unnecessary because https://github.com/react-native-video/react-native-video/blob/a79c30eaba277db4cd13b3515567c17b7a09ee58/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java#L1012 will call show() when `STATE_READY`

Reply:
OK for me thanks